Makefile

```diff
@@ -13,20 +13,20 @@ phpunit:
 phpdoc:
 	phpdoc run --no-interaction
 
-phpcs:
+initphpcs:
 	test -f /tmp/phpcs || curl -L https://squizlabs.github.io/PHP_CodeSniffer/phpcs.phar -o /tmp/phpcs && chmod +x /tmp/phpcs
 	test -d /tmp/wpcs || git clone -b master --depth 1 https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git /tmp/wpcs
```

Comment:
WPCS `master` and VIPCS `master` are not yet compatible.

The safest option for now, is to checkout the `1.2.1` tag of WPCS, and the `0.4.0` tag of VIPCS.

I'll make it clear in VIP when a release of VIPCS is compatible with WPCS `master`.

Reply:
VIPCS 2.0 has been released, and it is compatible with WPCS 2.1+.